cloudigrade/account/tests/helper.py

```diff
@@ -0,0 +1,105 @@
+"""Helper functions for generating test data."""
+import random
+import uuid
+
+from account.models import Account, Instance, InstanceEvent
+from util import aws
+from util.tests import helper
+
+
+def generate_account(arn=None, account_id=None):
+    """
+    Generate an Account for testing for random ARN and AWS Account ID.
+
+    Args:
+        arn (str): Optional ARN for the Account.
+        account_id (decimal.Decimal): Optional AWS Account ID.
+
+    Returns:
+        Account: The created Account.
+
+    """
+    if arn is None:
+        arn = helper.generate_dummy_arn(account_id)
+    if account_id is None:
+        account_id = aws.extract_account_id_from_arn(arn)
+    account = Account(account_arn=arn, account_id=account_id)
+    return account
+
+
+def generate_instance(account, ec2_instance_id=None, region=None):
+    """
+    Generate an Instance for the Account for testing with random attributes.
+
+    Args:
+        account (Account): Account that owns the Instance.
+        ec2_instance_id (str): Optional EC2 Instance ID.
+        region (str): Optional AWS region where the Instance runs.
+
+    Returns:
+        Instance: The created Instance.
+
+    """
+    if ec2_instance_id is None:
+        ec2_instance_id = str(uuid.uuid4())
+    if region is None:
+        region = random.choice(helper.SOME_AWS_REGIONS)
+
+    return Instance(
+        account=account,
+        ec2_instance_id=ec2_instance_id,
+        region=region
+    )
+
+
+def generate_instance_events(instance, powered_times, ec2_ami_id=None,
```

Comment:
`powered_times` here referred to as `powerd_on_times` in docstring.

Reply:
good 👀.